Podfile.lock

```diff
@@ -37,12 +37,12 @@ SPEC REPOS:
 
 SPEC CHECKSUMS:
   Expecta: 3b6bd90a64b9a1dcb0b70aa0e10a7f8f631667d5
-  InAppSettingsKit: 01fb9774c74a11fefc521f29cb2338909e1c2b61
-  OCMock: 75fbeaa46a9b11f8c182bbb1d1f7e9a35ccc9955
-  OHHTTPStubs: cb29d2a9d09a828ecb93349a2b0c64f99e0db89f
-  Specta: 3e1bd89c3517421982dc4d1c992503e48bd5fe66
-  xcbeautify: a3b03e4a38eb1a5766a83a7a3c53915a233572e3
+  InAppSettingsKit: 37df0b44132380d4c7db6fc7cded92997e29873a
+  OCMock: 9491e4bec59e0b267d52a9184ff5605995e74be8
+  OHHTTPStubs: 90eac6d8f2c18317baeca36698523dc67c513831
+  Specta: b79d84043684b35ffdc2680df578dc318ec2efc2
+  xcbeautify: 663b4c0c6e1ba22b5dfaf9382202473d3bd52046
 
 PODFILE CHECKSUM: 75163f16229528991a9364c7c1a44cd57a30cac6
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.2
```

Comment:
👋 We probably don't want to downgrade our CocoaPods version.

_For some background, we use this Podfile for internal dependencies on our Demo app and unit test suite. We also use [Bundler](https://github.com/braintree/braintree_ios/blob/master/Gemfile.lock) to manage our CocoaPods versions, so that each dev contributing is using the same version of CocoaPods (`bundle exec pod install`)._

For the purpose of this PR - I think we can just not commit the changes to this file and can handle bumping some of our internal pod dependencies in another PR.